### PR TITLE
Chore/137256077/deploy app to heroku

### PR DESCRIPTION
**Pull request by otoloye**

### CODE REVIEW

File: config/env/all.js [hunk 1]
```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-    db: process.env.MONGOHQ_URL
+    db: process.env.MONGOHQ_URL || 'mongodb://localhost:27017'
```
Comment (q-ode) [inline config/env/all.js]: What are you trying to achieve with this: `|| 'mongodb://localhost:27017'`?

File: app/views/includes/foot.jade [hunk 2]
```diff
@@ -10,9 +10,9 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 script(type='text/javascript', src='/lib/bootstrap/js/bootstrap.js')
 
 //AngularJS
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular.js')
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-resource.js')
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-cookies.js')
+script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular.js')
```
Comment (nyambati) [inline app/views/includes/foot.jade:13]: These files should be downloaded using bower instead of using CDN

**Comment by andela-ooyeniran** [comment]
I assigned/specified a port for it to run locally on my computer, am I suppose to take it out when I deploy?

File: config/env/all.js (hunk 1 above)
Comment (otoloye) [reply config/env/all.js]: I assigned/specified a port for it to run locally on my computer, am I suppose to take it out when I deploy?

Comment (q-ode) [reply config/env/all.js]: What I see you're doing is specifying the MongoDB URL for the database as `mongodb://localhost:27017` if it doesn't exist in the environment variables.

I think you could specify this URL in the `MONGOHQ_URL` environment variable for this project.

Checkout the `dotenv` module.

Comment (otoloye) [reply config/env/all.js]: ok thanks

**Review by nyambati — CHANGES_REQUESTED** [review]
Follow this guide to configure a correct deployment to Heroku and make the necessary changes required for a successful automated deploy.

https://docs.travis-ci.com/user/deployment/heroku/

**Review by nyambati — APPROVED** [review]

**Pull request merged**